data-prepper-plugins/saas-source-plugins/saas-crawler/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
```

Comment:
You don't need these lines.

Reply:
I thought we need these line when it is a library. I don't see any issue after removing them so removed 👍